evidence.go

```diff
@@ -82,19 +79,20 @@ func (e *Evidence) Sign(signer *cose.Signer) ([]byte, error) {
 		return nil, err
 	}
 
-	alg := signer.GetAlg()
-	if alg == nil {
+	alg := (*signer).Algorithm()
```

Comment:
I guess we can change the signature to `fun (e *Evidence) Sign(signer cose.Signer) ...`

Reply:
Valid point, I was thinking about that and I have now done it!